flox-bash/lib/commands/publish.sh

```diff
@@ -459,7 +459,7 @@ function floxPublish() {
 	else
 		gitClone="$tmpdir"
 		warn "Cloning $channelRepository ..."
-		$invoke_gh repo clone "$channelRepository" "$gitClone"
+		$invoke_flox_gh repo clone "$channelRepository" "$gitClone"
```

Comment:
```suggestion
		$invoke_gh repo clone "$channelRepository" "$gitClone"
```
Accidental change .. this needs to be the real `gh`.